telemetry/vscode/tst/generatorTest.ts

```diff
@@ -0,0 +1,40 @@
+/*!
```

Comment:
nit (naming): why "tst" instead of "test"?

- in the [jetbrains case](https://github.com/aws/aws-toolkit-common/tree/73e05585f323cea9f17d19b9d3f88f3209d4b7cd/telemetry/jetbrains/src/test) the test files live in `src/test/`
- in https://github.com/aws/aws-toolkit-vscode tests live in `src/test/`

As a habit it is good to avoid entropy if there is not a strong reason for the entropy.

Reply:
It's different because I'm bad at naming things consistently, it was not a conscious change. fixed